composer.json

```
@@ -41,7 +41,7 @@
         "php": ">=5.6",
         "illuminate/container": "~5.1|^6.0|^7.0|^8.0",
         "mnapoli/silly": "~1.5.0",
-        "symfony/process": "~2.7|~3.0|~4.0|~5.0",
+        "symfony/process": "^3.0",
```

To fix #587, this need to be version 4.2+

```suggestion
        "symfony/process": "^4.2",
```